x-pack/plugins/observability_solution/inventory/public/components/inventory_page_template/index.tsx

```diff
@@ -16,11 +22,30 @@ import { Welcome } from '../entity_enablement/welcome_modal';
 import { useInventoryAbortableAsync } from '../../hooks/use_inventory_abortable_async';
 import { EmptyState } from '../empty_states/empty_state';
 
-const pageTitle = {
-  pageTitle: i18n.translate('xpack.inventory.inventoryPageHeaderLabel', {
-    defaultMessage: 'Inventory',
-  }),
-};
+const pageTitle = (
+  <EuiFlexGroup gutterSize="s">
+    <EuiFlexItem grow={false}>
+      {i18n.translate('xpack.inventory.inventoryPageHeaderLabel', {
+        defaultMessage: 'Inventory',
+      })}
+    </EuiFlexItem>
+    <EuiFlexItem grow={false}>
+      <EuiBetaBadge
```

Comment:
You can reuse the TP badge from the oblt share plugin - https://github.com/elastic/kibana/blob/37342c882b55f59154ef3440947b306ffbe54f26/x-pack/plugins/observability_solution/observability_shared/public/components/technical_preview_badge/technical_preview_badge.tsx#L16